src/components/home-address/Header.tsx

```diff
@@ -0,0 +1,40 @@
+import { COLORS } from "@/constants/css";
+import { ColorProps } from "@/types/css-props";
+import styled from "@emotion/styled";
+import Image from "next/image";
+
+const HEADER_TEXT = "어디서 만날까?";
+
+const Header = () => {
+  return (
+    <HeaderContainer colors={COLORS}>
+      <TextP>{HEADER_TEXT}</TextP>
+      <Image
+        src='/Logo.png'
+        alt='Odeego Logo'
+        width={137}
+        height={46}
+        priority
+      />
+    </HeaderContainer>
+  );
+};
+
+export default Header;
+
+const HeaderContainer = styled.header<{ colors: ColorProps }>`
+  min-height: 174px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  background-color: ${({ colors }) => colors.backgroundAccent};
```

Comment:
'colors'를 prop으로 받아쓰는 방법보다 바로 'COLORS' 변수를 가져다 사용할 수 있을것같아요!
```suggestion
  background-color: ${COLORS.backgroundAccent};
```

Reply:
음? 반영했는데 안 들어갔군요? 이상하네요...:)?